model/resource/k8s.yaml

```
@@ -85,6 +85,11 @@ groups:
         brief: >
           The name of the Pod.
         examples: ['opentelemetry-pod-autoconf']
+      - id: labels
+        type: template[string]
+        brief: >
+          The labels placed on the Pod, the `<key>` being the label name, the value being the label value.
+        examples: ['pod.labels.app=my-app,pod.labels.mycompany.io/arch=x64']
```

```suggestion
        examples: ['k8s.pod.labels.app=my-app,pod.labels.mycompany.io/arch=x64']
```
In other places like `http` and `containers` the examples come with the full prefix:
1) https://github.com/open-telemetry/semantic-conventions/blob/f6fec98892f1b7a1e92b2cfff0812c40f6195c1e/model/registry/http.yaml#L30
2) https://github.com/open-telemetry/semantic-conventions/blob/f6fec98892f1b7a1e92b2cfff0812c40f6195c1e/model/registry/container.yaml#L86

---

Good point, thank you. Fixed for both labels in the example, and added a third label to demonstrate that it could be empty.